src/main/steps/applicant1/their-email-address/content.ts

```diff
@@ -39,9 +40,10 @@ const en = ({ partner, isDivorce, isJointApplication, hasEnteredSolicitorDetails
       sameEmail: `You have entered your own email address. You need to enter your ${partner}'s email address before continuing.`,
     },
   },
+  continueOrResend: isApplicant2EmailUpdatePossible(userCase) ? 'Resend email' : 'Continue',
 });
 
-const cy: typeof en = ({ partner }) => ({
+const cy = ({ userCase, partner }) => ({
```

Comment:
worth keeping in 
const cy: typeof en = ({ partner }) => ({

Reply:
done